x/stakeibc/keeper/validator_selection_test.go

```diff
@@ -1,18 +1,159 @@
 package keeper_test
 
 import (
+	"fmt"
 	"math/rand"
 
 	sdkmath "cosmossdk.io/math"
+	"github.com/gogo/protobuf/proto" //nolint:staticcheck
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	ibctesting "github.com/cosmos/ibc-go/v5/testing"
+
 	// TODO [LSM]: Revert type
```

Comment:
What is this comment about?

Reply:
its cause we had to change to proto types to the ones from LSM